app_base.htm

```diff
@@ -0,0 +1,58 @@
+<%inherit file="/homepage/templates/base.htm" />
+<%! from catalog import models as cmod %>
+
+<%block name="left">
+  <ul class="category_list">
+    <li><a class="${ 'active' if cat_name == 'All Products' else '' }" href="/catalog/index/0">All Products</a></li>
+
+    % for cat in cmod.Category.objects.all():
+      <li><a class="${ 'active' if cat_name == cat.name else '' }" href="/catalog/index/${ cat.id }">${ cat.name }</a></li>
+    % endfor
+  </ul>
+</%block>
+
+<%block name="menu">
+  <div class="collapse navbar-collapse">
+      <ul class="nav navbar-nav nav-pills">
+        <li class="${ 'active' if request.dmp.page == '8000/index' else '' }"><a href="/">Home</a></li>
```

Comment:
Not sure you need the 8000 in the request here

Reply:
There were two index. So I needed to distinguish them :)